__tests__/app.test.js

```diff
@@ -323,14 +323,71 @@ describe(`/api/articles`, () => {
         });
     });
 
-    test("response should be sorted by date in descending order", () => {
+    test("response should have default ordering by date and default sorting of descending", () => {
       return request(app)
         .get("/api/articles")
         .then(({ body: { articles } }) => {
           expect(articles).toBeSortedBy("created_at", { descending: true });
         });
     });
 
+    test("should accept query to sort by specific column", () => {
+      return request(app)
+        .get("/api/articles?sort=votes")
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeSortedBy("votes", { descending: true });
+        });
+    });
+
+    test("should accept query to specify ordering type", () => {
+      return request(app)
+        .get("/api/articles?order=asc")
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeSortedBy("created_at", { ascending: true });
+        });
+    });
+
+    test("should accept query to filter response by article topic", () => {
+      const query = "mitch";
+      return request(app)
+        .get(`/api/articles?topic=${query}`)
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeSortedBy("created_at", { descending: true });
+          expect(articles).toEqual(
+            expect.arrayContaining([expect.objectContaining({ topic: query })])
+          );
+        });
+    });
+
+    test("should accept more than one query at a time", () => {
+      return request(app)
+        .get("/api/articles?sort=votes&order=asc")
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeSortedBy("votes", { ascending: true });
+        });
+    });
+
+    test("should accept all three queries at the same time", () => {
+      const query = "mitch";
+      return request(app)
+        .get(`/api/articles?sort=votes&topic=${query}&order=asc`)
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeSortedBy("votes", { ascending: true });
+          expect(articles).toEqual(
+            expect.arrayContaining([expect.objectContaining({ topic: query })])
```

Comment:
again would want you to check the whole array of articles all contains the right topic, at the moment you're checking that at least one article has it